Client/core/CCore.h

```diff
@@ -285,6 +285,10 @@ class CCore : public CCoreInterface, public CSingleton<CCore>
     void   SetCustomStreamingMemory(size_t szMB);
     bool   IsUsingCustomStreamingMemorySize();
     size_t GetStreamingMemory();
+
+    SString GetLastConnectedServerName() const { return m_strLastConnectedServerName; }
+    void    SetLastConnectedServerName(SString strServerName) { m_strLastConnectedServerName = strServerName; }
```

Comment:
Unnecessary string copy when passing it as an argument. Can be eliminated by using a const reference or move semantics.